13-master/Dockerfile

```diff
@@ -327,5 +326,11 @@ RUN set -ex \
 RUN mkdir -p /docker-entrypoint-initdb.d
 COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/10_postgis.sh
 COPY ./update-postgis.sh /usr/local/bin
-RUN cat /_pgis_full_version.txt
 
+RUN set -ex \
```

Comment:
This only queries to see if the ca-certificates package is installed and reports the result, correct?  If the package is not installed, there's no place I see where the package gets installed.